darts/models/forecasting/transformer_model.py

```diff
@@ -237,7 +252,9 @@ def __init__(
         dropout
             Fraction of neurons affected by Dropout (default=0.1).
         activation
-            The activation function of encoder/decoder intermediate layer, 'relu' or 'gelu' (default='relu').
+            The activation function of encoder/decoder intermediate layer, (default='relu').
+            can be one of the glu variant's FFN ["GLU", "Bilinear", "ReGLU", "GEGLU", "SwiGLU", "ReLU", "GELU"]
```

Comment:
Can you maybe refer to the paper here, and explain in a couple of sentences what is FFN?

Also: how about optionally adding these to the TFT model which is also transformer based?

Reply:
done